ReactAndroid/src/main/java/com/facebook/react/uimanager/ReactClippingViewGroup.java

```diff
@@ -7,13 +7,11 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  */
 
-package com.facebook.react.views.view;
+package com.facebook.react.uimanager;
```

Comment:
Was there a reason this was moved? There are internal classes in FB that are referencing this class and that's what's causing this patch to fail to land. It'd be easiest if we don't move this if we don't need to.